cli/cmd/get_event_test.go

```diff
@@ -15,18 +13,6 @@ func init() {
 	logging.InitLoggers(os.Stdout, os.Stdout, os.Stderr)
 }
 
-func TestGetEventCmdEmptyInput(t *testing.T) {
-	credentialmanager.MockAuthCreds = true
-
-	cmd := fmt.Sprintf("get event --project=%s",
-		"sockshop")
-	_, err := executeActionCommandC(cmd)
-
-	if !errorContains(err, "please provide an event type as an argument") {
-		t.Errorf("missing expected error, but got %v", err)
-	}
-}
-
```

Comment:
Why was this test removed? Does it no longer make sense or can we query all events without a type now?

Reply:
yes, the new implementation allows passing no argument. In this case the result will just not be filtered by the event type.The test makes no sense anymore